### PR TITLE
Support TSxxx commands for communicating with YugaByteDB

### PR DESCRIPTION
**Pull request by amitanandaiyer**

### CODE REVIEW

**Comment by bbaddepudi** [comment]
Would be good to also update the API info in the README, @amitanandaiyer. Thanks.

File: redis/client.py [hunk 1]
```diff
@@ -1675,6 +1683,64 @@ def sunionstore(self, dest, keys, *args):
...
+                raise RedisError("TSADD requires an equal number of "
+                                 "values and scores")
+            pieces.extend(args)
+        for pair in iteritems(kwargs):
+            pieces.append(pair[1])
+            pieces.append(pair[0])
+        return self.execute_command('TSADD', name, *pieces)
+
+    def tsget(self, name, ts):
+        "Return the element corresponding to the specified timestamp `ts` in the time series``name``"
```
Comment (bbaddepudi) [inline redis/client.py]: Might be good to keep the same comment style - use """ encompassing the contents.

File: README.rst [hunk 2]
```diff
@@ -10,6 +10,14 @@ The Python interface to the Redis key-value store.
 .. image:: https://badge.fury.io/py/redis.svg
         :target: https://pypi.org/project/redis/
 
+This has been forked from https://github.com/andymccurdy/redis-py and is intended to be
+used against the Yedis API end point of YugaByteDB. This client also supports additional
+commands (such as the time-series related commands) for use againist YugaByte DB that
+are not part of the official Redis API.
+
+Please refer to `YugaByte DB docs <https://docs.yugabyte.com/>`_ for reference.
```
Comment (bbaddepudi) [inline README.rst:18]: Is there an extra _ after the url? If so, can be removed...

File: README.rst [hunk 3]
```diff
@@ -10,6 +10,14 @@ The Python interface to the Redis key-value store.
 .. image:: https://badge.fury.io/py/redis.svg
         :target: https://pypi.org/project/redis/
 
+This has been forked from https://github.com/andymccurdy/redis-py and is intended to be
+used against the Yedis API end point of YugaByteDB. This client also supports additional
+commands (such as the time-series related commands) for use againist YugaByte DB that
```
Comment (bbaddepudi) [inline README.rst:15]: nit: against

**Review by bbaddepudi — APPROVED** [review]
Couple of suggestions, LG after that from my side. Thanks.

File: README.rst (hunk 2 above)
Comment (amitanandaiyer) [reply README.rst:18]: Tried removing it. Looks like we need it for the anchor text to be seen properly. Adding it back.

**Pull request merged**